neo/Ledger/PoolItem.cs

```diff
@@ -0,0 +1,45 @@
+using Neo.Network.P2P.Payloads;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using Akka.Util.Internal;
+using Neo.Network.P2P;
+using Neo.Persistence;
+using Neo.Plugins;
+
+namespace Neo.Ledger
+{
+    public class PoolItem : IComparable<PoolItem>
```

Comment:
This should not be public. If we are going to do this it should be internal. But unit tests already test this order now, so I don't see a need to do this really.

Reply:
Perhaps I missed the tests for PoolItem then, because I was trying to understand how two were actually compared. I didn't see an explicit case for that, and I think it's important.
If it's internal, perhaps it's better not to create a file for it... don't know C# standards well, but makes sense for me.

Reply:
The output of MemoryPool now tests all the orders that things occur. If Sort order of PoolItem were wrong in any way the tests for MemoryPool now cover it and would fail.

Reply:
Maybe we should do something though to ensure now that there are always two items inserted in the tests that have the same fee. So the sorted test is a bit better. I guess that is one advantage of pool item tests, but we could also just make these ones better.

Reply:
That is it.

Reply:
I'm fine with moving this out as long as it is made `internal` and you can add whatever tests you want, though sort order ones may be redundant.

Reply:
Better to be more redundant and avoid other errors, Jeff.
Because in one hand we check the compareTo and on the other hand we can check the MemPool SortedSet and its logic.